spec/electra/executionpayload_yaml.go

```diff
@@ -25,25 +25,26 @@ import (
 
 // executionPayloadYAML is the spec representation of the struct.
 type executionPayloadYAML struct {
-	ParentHash         string                             `yaml:"parent_hash"`
-	FeeRecipient       string                             `yaml:"fee_recipient"`
-	StateRoot          string                             `yaml:"state_root"`
-	ReceiptsRoot       string                             `yaml:"receipts_root"`
-	LogsBloom          string                             `yaml:"logs_bloom"`
-	PrevRandao         string                             `yaml:"prev_randao"`
-	BlockNumber        uint64                             `yaml:"block_number"`
-	GasLimit           uint64                             `yaml:"gas_limit"`
-	GasUsed            uint64                             `yaml:"gas_used"`
-	Timestamp          uint64                             `yaml:"timestamp"`
-	ExtraData          string                             `yaml:"extra_data"`
-	BaseFeePerGas      string                             `yaml:"base_fee_per_gas"`
-	BlockHash          string                             `yaml:"block_hash"`
-	Transactions       []string                           `yaml:"transactions"`
-	Withdrawals        []*capella.Withdrawal              `yaml:"withdrawals"`
-	BlobGasUsed        uint64                             `yaml:"blob_gas_used"`
-	ExcessBlobGas      uint64                             `yaml:"excess_blob_gas"`
-	DepositReceipts    []*DepositReceipt                  `yaml:"deposit_receipts"`
-	WithdrawalRequests []*ExecutionLayerWithdrawalRequest `yaml:"withdrawal_requests"`
+	ParentHash            string                  `yaml:"parent_hash"`
+	FeeRecipient          string                  `yaml:"fee_recipient"`
+	StateRoot             string                  `yaml:"state_root"`
+	ReceiptsRoot          string                  `yaml:"receipts_root"`
+	LogsBloom             string                  `yaml:"logs_bloom"`
+	PrevRandao            string                  `yaml:"prev_randao"`
+	BlockNumber           uint64                  `yaml:"block_number"`
+	GasLimit              uint64                  `yaml:"gas_limit"`
+	GasUsed               uint64                  `yaml:"gas_used"`
+	Timestamp             uint64                  `yaml:"timestamp"`
+	ExtraData             string                  `yaml:"extra_data"`
+	BaseFeePerGas         string                  `yaml:"base_fee_per_gas"`
+	BlockHash             string                  `yaml:"block_hash"`
+	Transactions          []string                `yaml:"transactions"`
+	Withdrawals           []*capella.Withdrawal   `yaml:"withdrawals"`
+	BlobGasUsed           uint64                  `yaml:"blob_gas_used"`
+	ExcessBlobGas         uint64                  `yaml:"excess_blob_gas"`
+	DepositRequests       []*DepositRequest       `yaml:"deposit_receipts"`
```

Comment:
Again, should be `deposit_requests`?

Reply:
fixed & found even more places where it was still called `deposit_receipts`